internal/resource/configmap_test.go

```diff
@@ -235,7 +222,7 @@ CONSOLE_LOG=new`
 				Expect(configMap.Data).ToNot(HaveKey("rabbitmq-env.conf"))
 			})
 
-			Context("rabbitmq-env.conf is set", func() {
+			Context("rabbitmq-env.conf i set", func() {
```

Comment:
You probably didn't mean to change this line

Reply:
Oh yes, thanks for spotting that.